provisioning-workflows/zOS/NetworkResources/Deprovision_Network_Resource.xml

```diff
@@ -0,0 +1,182 @@
+                        <targetState>Skipped</targetState>
+                    </extendStateExpression>
+                    <extendStateExpression>
+                        <description>Skipped if PORT_ID is null string</description>
+                        <expression> ${instance_PORT_ID} == ""</expression>
+                        <targetState>Skipped</targetState>
+                    </extendStateExpression>
+                </targetStateSet>
+            </condition>
+            <instructions substitution="false">
+            Execute step to return the HTTP port back to z/OS Communications Server.
+            </instructions>
+            <weight>10</weight>
+            <skills>REST</skills>
+            <autoEnable>true</autoEnable>
+            <canMarkAsFailed>false</canMarkAsFailed>
+            <rest>
+                <httpMethod>POST</httpMethod>
+                <uriPath substitution="true">/zosmf/resource-mgmt/rest/1.0/rdp/network/port/actions/release</uriPath>
+                <requestBody substitution="true">
+
+                    {
+
+                    "template-uuid" : "${_workflow-templateID}",
+                    "template-name" : "${_workflow-templateName}",
+                    "tenant-id" : "${_workflow-tenantID}",
+                    "network-parms" :
+                            {
+                            "port-id" : "${instance-PORT_ID}"
+                            }
+                    }
+
+                </requestBody>
+                <expectedStatusCode>204</expectedStatusCode>
+            </rest>
+            <setVariable name="PORT_ID" scope="instance"></setVariable>
+        </step>
+        <step name="returnAPPLID" optional="false">
+            <title>Return APPLID</title>
+            <description>Invoke RESTful API to return APPLID back to pool.</description>
+            <condition>
+                <expression>"1" == "1"</expression>
+                <description>Check that previous step has completed</description>
+                <targetStateSet>
+                    <description>Set state to Skipped if APPLID_ID is not set.</description>
+                    <extendStateExpression>
+                        <description>Skipped if APPLID_ID is null</description>
+                        <expression> ${instance_APPLID_ID} == null</expression>
+                        <targetState>Skipped</targetState>
+                    </extendStateExpression>
+                    <extendStateExpression>
+                        <description>Skipped if APPLID_ID is null string</description>
+                        <expression> ${instance_APPLID_ID} == ""</expression>
+                        <targetState>Skipped</targetState>
+                    </extendStateExpression>
+                </targetStateSet>
+            </condition>
+            <instructions substitution="false">
+                 Execute step to return the VTAM APPLID back to z/OS Communications Server.
```

Comment:
VTAM?  I see VTAM is being used by other workflows in this commit, but this is the only time this phrase is used during deprovisioning. Should it be used in other locations as well, or omitted here?